extensions-contrib/parquet-extensions/example/parser/events_with_all_datatype_test_combinations.json

```diff
@@ -0,0 +1,240 @@
+              "dimensionName": "tag_ids"
+            },
+            {
+              "key": "tags",
+              "fieldType": "LIST",
+              "field": {
+                "key": "tagCostAttributes",
+                "fieldType": "STRUCT",
+                "field": {
+                  "key": "CostA",
+                  "fieldType": "MAP",
+                  "rootFieldName": "tagCostAttributes"
+                }
+              },
+              "dimensionName": "cost_a"
+            },
+            {
+              "key": "tags",
+              "fieldType": "LIST",
+              "index": 0,
+              "field": {
+                "key": "CostB",
+                "fieldType": "MAP",
+                "rootFieldName": "tagCostAttributes"
+              },
+              "dimensionName": "cost_b_1st_tag"
+            },
+            {
+              "key": "tags",
+              "fieldType": "LIST",
+              "index": 1,
+              "field": {
+                "key": "CostB",
+                "fieldType": "MAP",
+                "rootFieldName": "tagCostAttributes"
+              },
+              "dimensionName": "cost_b_2nd_tag"
+            }
+          ]
+        },
+        "parseSpec": {
+          "format": "timeAndDims",
+          "timestampSpec": {
+            "column": "eventTimestamp",
+            "format": "millis"
+          },
+          "dimensionsSpec": {
+            "dimensions": [
+              "eventId",
+              "valid_map_access",
+              "max_access_with_non_existing_key",
+              "max_access_with_un_available_map",
+              "array_access_with_index",
+              "array_access_with_default_index",
+              "array_access_with_overbound_index",
+              "union_attribute",
+              "tag_ids",
+              "reference_ids",
+              "non_available_list",
+              "list_with_non_available_index",
+              "cost_a",
+              "cost_b_1st_tag"
+            ],
+            "dimensionExclusions": [
+            ],
+            "spatialDimensions": [
+            ]
+          }
+        }
+      },
+      "metricsSpec": [
+        {
+          "type": "count",
+          "name": "count"
+        },
+        {
+          "type": "doubleSum",
+          "name": "cost_b_2nd_tag_sum",
+          "fieldName": "cost_b_2nd_tag"
+        }
+      ],
+      "granularitySpec": {
+        "type": "uniform",
+        "segmentGranularity": "DAY",
+        "queryGranularity": "DAY",
+        "rollup": true,
+        "intervals": [
+          "2017-01-26T00:00:00.000Z/2017-01-27T00:59:59.000Z"
+        ]
+      }
+    },
+    "ioConfig": {
+      "type": "hadoop",
+      "inputSpec": {
+        "type": "static",
+        "inputFormat": "io.druid.data.input.parquet.DruidParquetInputFormat",
+        "paths": "example/events/events_with_all_datatypes.parquet"
+      },
+      "segmentOutputPath": "/tmp/segments"
+    },
+    "tuningConfig": {
```

Comment:
For this example spec and others in the PR, I would suggest dummying out the contents of `tuningConfig` here, e.g.

```
"tuningConfig": {
...
}
```

since this section isn't really relevant to the parquet parsing and each deployment would have its own mapreduce properties and such.

I would also remove any references to a specific hadoop version such as 2.6.0.

Reply:
okay, we will look into it and remove all such instances.